Type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols

Following compilation errors prevent Gala 0.3.3 from building with Xcode 11.6:
```
Gala/Sources/Gala/Gala.swift:28:5: error: type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:28:5: note: required by referencing initializer 'init(_:id:content:)' on 'ForEach' where 'Content' = '()'
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:46:5: error: type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:46:5: note: required by referencing initializer 'init(_:id:content:)' on 'ForEach' where 'Content' = '()'
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:70:5: error: type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols
    ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:70:5: note: required by referencing initializer 'init(_:id:content:)' on 'ForEach' where 'Content' = '()'
    ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
    ^
```

--- a/Sources/Gala/Gala.swift
+++ b/Sources/Gala/Gala.swift
@@ -25,17 +25,19 @@ public func Layouts<A: View>(_ layouts: [Layout], _ name: String? = nil, @ViewBu
 @available(tvOS, unavailable)
 @available(watchOS, unavailable)
 public func HorizontalSizeClasses<A: View>(_ name: String? = nil, @ViewBuilder items: @escaping () -> A) -> some View {
-    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
-        #if swift(<5.3)
+    #if swift(<5.3)
+    return ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item!)" } ?? "\(item!)")
             .environment(\.horizontalSizeClass, item)
-        #else
+    }
+    #else
+    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item)" } ?? "\(item)")
             .environment(\.horizontalSizeClass, item)
-        #endif
     }
+    #endif
 }
 
 
@@ -43,17 +45,19 @@ public func HorizontalSizeClasses<A: View>(_ name: String? = nil, @ViewBuilder i
 @available(tvOS, unavailable)
 @available(watchOS, unavailable)
 public func VerticalSizeClasses<A: View>(_ name: String? = nil, @ViewBuilder items: @escaping () -> A) -> some View {
-    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
-        #if swift(<5.3)
+    #if swift(<5.3)
+    return ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item!)" } ?? "\(item!)")
             .environment(\.verticalSizeClass, item)
-        #else
+    }
+    #else
+    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item)" } ?? "\(item)")
             .environment(\.verticalSizeClass, item)
-        #endif
     }
+    #endif
 }
 
 
@@ -67,17 +71,19 @@ public func LayoutDirections<A: View>(_ name: String? = nil, @ViewBuilder items:
 
 
 public func LegibilityWeights<A: View>(_ name: String? = nil, @ViewBuilder items: @escaping () -> A) -> some View {
-    ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
-        #if swift(<5.3)
+    #if swift(<5.3)
+    return ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item!)" } ?? "\(item!)")
             .environment(\.legibilityWeight, item)
-        #else
+    }
+    #else
+    ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item)" } ?? "\(item)")
             .environment(\.legibilityWeight, item)
-        #endif
     }
+    #endif
 }
 
 public func ContentSizeCategories<A: View>(_ name: String? = nil, @ViewBuilder items: @escaping () -> A) -> some View {

--- a/Sources/Gala/Gala.swift
+++ b/Sources/Gala/Gala.swift
@@ -32,7 +32,7 @@ public func HorizontalSizeClasses<A: View>(_ name: String? = nil, @ViewBuilder i
             .environment(\.horizontalSizeClass, item)
     }
     #else
-    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
+    return ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item)" } ?? "\(item)")
             .environment(\.horizontalSizeClass, item)
@@ -52,7 +52,7 @@ public func VerticalSizeClasses<A: View>(_ name: String? = nil, @ViewBuilder ite
             .environment(\.verticalSizeClass, item)
     }
     #else
-    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
+    return ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item)" } ?? "\(item)")
             .environment(\.verticalSizeClass, item)
@@ -78,7 +78,7 @@ public func LegibilityWeights<A: View>(_ name: String? = nil, @ViewBuilder items
             .environment(\.legibilityWeight, item)
     }
     #else
-    ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
+    return ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
         items()
             .previewDisplayName(name.map { "\($0) \(item)" } ?? "\(item)")
             .environment(\.legibilityWeight, item)

